perfume-api/src/main/java/io/perfume/api/sample/application/SampleService.java

```diff
@@ -0,0 +1,68 @@
+package io.perfume.api.sample.application;
+
+import io.perfume.api.sample.application.dto.SampleResult;
+import io.perfume.api.sample.application.exception.AccessDeniedException;
+import io.perfume.api.sample.application.exception.EntityNotFoundException;
+import io.perfume.api.sample.application.port.SampleQueryRepository;
+import io.perfume.api.sample.application.port.SampleRepository;
+import io.perfume.api.sample.domain.Sample;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class SampleService {
+
+    private final SampleRepository sampleRepository;
+    private final SampleQueryRepository sampleQueryRepository;
+
+    @Transactional(readOnly = true)
+    public List<SampleResult> getSamples() {
+        return sampleQueryRepository.find().stream().map(this::toDto).toList();
+    }
+
+    @Transactional(readOnly = true)
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.